application/config.py

```diff
@@ -0,0 +1,29 @@
+class Config(object):
+    DEBUG = False
+    CSRF_ENABLED = False
+    SECRET = 'dcfgyhbnjio9uytrfcfr4edfg'
+    SQLALCHEMY_DATABASE_URI = 'postgresql://localhost/som_timer'
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+class DevelopmentConfig(Config):
+    DEBUG = True
+    BUNDLE_ERRORS = True
+
+
+class TestingConfig(Config):
+    TESTING = True
+    DEBUG = True
```

Comment:
Still have yet to play around enough with debugger, but I'm hoping it works like 'pry' in Ruby!